llvm/lib/Transforms/Vectorize/VPlan.cpp

```diff
@@ -920,6 +979,10 @@ void VPlan::execute(VPTransformState *State) {
          "DT not preserved correctly");
 }
 
+InstructionCost VPlan::cost(ElementCount VF, VPCostContext &Ctx) {
```

Comment:
May be worth a note that only the loop region is measured, preheader and post exit recipes are neglected.